### DISCUSSION TITLE
Кнопка перевести видео то появляется не с первого раза.

### DISCUSSION THREAD

**Discussion by ghost** [ilyhalight/voice-over-translation#368] [Q&A]
Привет всем, я использую браузер Edge, но подобное встречается и на Firefox и на других. Суть в том, что очень часто заходя на видео, кнопка перевести видео не появляется, однако если зайти на видео ещё раз, причём не через обновление страницы, а именно заного, то кнопка появляется. С чем это может быть связано и как с этим бороться?

**Comment by NullVerdict** [comment]
> Привет всем, я использую браузер Edge, но подобное встречается и на Firefox и на других. Суть в том, что очень часто заходя на видео, кнопка перевести видео не появляется, однако если зайти на видео ещё раз, причём не через обновление страницы, а именно заного, то кнопка появляется. С чем это может быть связано и как с этим бороться?

Попробуйте бета версию, возможно вам поможет.

**Comment by FiLinX** [comment]
> очень часто заходя на видео, кнопка перевести видео не появляется

да, такое поведение встречается, и было оно всегда.. не часто, но бывает
пока помогает просто перезагруз страниц
по моим наблюдениям, такой глюк чаще всего проявляется при открытии ссылок в фоновой вкладке, - что то в браузере рендер страниц нарушает, и я грешу лишь на нынешнее время - ну, постоянно происходят проблемы с днс и доступами к контенту сторонних ресурсов на страницах по tls 
на страницах тупят всякие там аватарки, смайлы, скрипты.. не успевают они вовремя отрендерится и их отлюпляет
при этом лупит и VOT
